data-model/digitalobjects/0.2.0/digital-specimens/schema/identifications.json

```diff
@@ -0,0 +1,365 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.2.0/digital-specimens/identifications.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.2.0",
+  "type": "object",
+  "properties": {
+    "dwc:identificationID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationID",
+      "examples": [
+        "9992"
+      ]
+    },
+    "???:identificationType": {
```

Comment:
Remarks is also in there, I got identificationType from https://github.com/gbif/model-material and it might be handy when we get non-taxonomic identification (for example chemical)